Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/pch.h

```diff
@@ -30,4 +30,4 @@
 #define NODE_ADDON_API_DISABLE_DEPRECATED
 #endif
 
-#include "../../shared/BabylonNative.h"
\ No newline at end of file
+#include "BabylonNative.h"
```

Comment:
Huh, I guess it was just an oversight that I didn't do this for iOS/Android? You didn't change anything to make this possible now where it previously wasn't right?

Reply:
the vcxproj did not have the shared folder added as an additional include path. now it does for easier reading